runexporterapp.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+FLASK_RUN_HOST=${FLASK_RUN_HOST:-"0.0.0.0"}
+FLASK_RUN_PORT=${FLASK_RUN_PORT:-"9608"}
+
+export IRONIC_CONFIG="/etc/ironic/ironic.conf"
+
+gunicorn -b ${FLASK_RUN_HOST}:${FLASK_RUN_PORT} -w 4 ironic_prometheus_exporter.app.wsgi:application
```

Comment:
Do we need `exec` here to remove the `bash` process from the picture?

Reply:
yes, good point